manifests/n/nimblenote/nimblenote/2.0.2/nimblenote.nimblenote.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher:leaves under foot ltd
```

```suggestion
Publisher: leaves under foot ltd
```
My bad
`(Line: 7, Col: 1, Idx: 223) - (Line: 7, Col: 32, Idx: 254): While parsing a block mapping, did not find expected key.`